frontend/javascripts/oxalis/view/left-border-tabs/modals/add_volume_layer_modal.tsx

```diff
@@ -48,7 +48,7 @@ export function checkLayerNameForInvalidCharacters(readableLayerName: string): V
       message: messages["tracing.volume_layer_name_starts_with_dot"],
     };
   }
-  const uriSafeCharactersRegex = /[0-9a-zA-Z-._]+/g;
+  const uriSafeCharactersRegex = /[0-9a-zA-Z-._$]+/g;
```

Comment:
If I remember correctly you found that these characters don't necessarily have to be url safe, right? If so, maybe rename to `validLayerNameCharactersRegex` or so instead.